Files/ViewModels/ItemViewModel.cs

```diff
@@ -1010,9 +1010,9 @@ public async Task<bool> EnumerateItemsFromStandardFolderAsync(string path)
                     {
                         var value = new ValueSet();
                         value.Add("Arguments", "Bitlocker");
-                        value.Add("action", "Unlock");
-                        value.Add("drive", Path.GetPathRoot(WorkingDirectory));
-                        value.Add("password", userInput);
+                        value.Add("Action", "Unlock");
```

Comment:
I think it's better to minimize changes outside the Cpp Fulltrustprocess unless there's a compelling reason, since it will be a long time before merging to master

Reply:
hmm... This is the only instance where Bitlocker call is used. Since I had a chance, I wanted to unify the names of parameters

Reply:
I think every other call uses uppercase for Arguments and lowercase for everything else